hicommonwealth/commonwealth
# [ERC20 Launchpad] - Store token info

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9371

## Description of Changes
Allows token information to be stored on the API side during tokenized community launch flow. 

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit `/createTokenCommunity`
3. Complete the token information step (if you have already created a token before, verify you get the "Token name already exists" error when using an existing token name) 
4. When redirected to the community information step, verify community information is pre-filled with token information provided earlier. Submit the form to continue with the signature signing step.
5. On the signature signing step, if you click cancel, then verify you get redirected to the success step but it only shows that you successfully created the community but not the token.
6. On the signature signing step, if you click sign, verify you get prompted to sign a transaction to launch the token on base sepolia, and on successfully signing the transaction you get redirected to the success step and it shows that you successfully created the token and community and clicking on 'Go to Community' takes you to the community discussion page.

## Deployment Plan
N/A

## Other Considerations
N/A
